docs/src/whatsnew/whatsnew.rst

```diff
@@ -1,11 +1,11 @@
+.. include:: ../common_links.inc
+
 .. _iris_whatsnew:
 
 What's New in Iris
-******************
-
-These "What's new" pages describe the important changes between major
-Iris versions.
+------------------
 
+.. include:: latest.rst
 
 .. toctree::
    :maxdepth: 1
```

Comment:
When the page is rendered the latest info is shown via a single click  but the toc is then also displayed at the bottom which doesn't quite look right imho.  As we have the toc in the left sidebar I think we could hide the toc in the main body by using the ``:hidden:`` option under the toctree directive, I think that will solve it.

Reply:
[aab13c4](https://github.com/SciTools/iris/pull/4727/commits/aab13c401ef69f3f359cbbcbfd9fbac11acf63ae)